istioctl/cmd/analyze.go

```diff
@@ -200,6 +238,7 @@ istioctl experimental analyze -k -d false
 		fmt.Sprintf("The severity level of analysis at which to set a non-zero exit code. Valid values: %v", diag.GetAllLevelStrings()))
 	analysisCmd.PersistentFlags().Var(&outputLevel, "output-threshold",
 		fmt.Sprintf("The severity level of analysis at which to display messages. Valid values: %v", diag.GetAllLevelStrings()))
+	analysisCmd.PersistentFlags().StringVarP(&msgOutputFormat, "output", "o", LogOutput, fmt.Sprintf("Output format: one of %v", msgOutputFormatKeys))
```

Comment:
Nit: I know we accept both, but let's have the usage be in terms of the lowercase versions, consistent with kubectl.